src/main/java/io/github/jhipster/online/service/GitlabService.java

```diff
@@ -110,7 +110,7 @@ public void syncUserFromGitProvider() throws Exception {
         if (user.isPresent()) {
             this.getSyncedUserFromGitProvider(user.get());
         } else {
-            log.info("No user was found to sync with GitLab ");
+            log.info("No user {} was found to sync with GitLab", user);
```

Comment:
Not sure this will be useful, I'd rather log current user login

    public void syncUserFromGitProvider() throws Exception {
        Optional<String> currentUserLogin = SecurityUtils.getCurrentUserLogin();
        Optional<User> user = userRepository.findOneByLogin(currentUserLogin.orElse(null));
        if (user.isPresent()) {
            this.getSyncedUserFromGitProvider(user.get());
        } else {
            log.info("No user {} was found to sync with GitLab", currentUserLogin);
        }
    }

Reply:
Thanks. Done. :+1: